src/components/DatePicker/index.js

```diff
@@ -25,8 +25,8 @@ class DatePicker extends React.Component {
     componentDidMount() {
         // Adds nice native datepicker on web/desktop. Not possible to set this through props
         this.inputRef.setAttribute('type', 'date');
-        this.inputRef.setAttribute('max', CONST.DATE.MAX_DATE);
-        this.inputRef.setAttribute('min', CONST.DATE.MIN_DATE);
+        this.inputRef.setAttribute('max', moment(this.props.maxDate).format(CONST.DATE.MOMENT_FORMAT_STRING));
+        this.inputRef.setAttribute('min', moment(this.props.minDate).format(CONST.DATE.MOMENT_FORMAT_STRING));
```

Comment:
Why do we need to format it here?

Reply:
For min and max attribute input type="date" accepts a formatted string (yyyy-mm-dd). We pass the prop in Date format, so it needs to be formatted to work properly. We do the same for the value and the defaultValue.